cg/meta/workflow/mutant.py

```diff
@@ -0,0 +1,154 @@
+            Customer_ID_sample=sample_obj.name,
+            customer_id=sample_obj.customer.internal_id,
+            sequencing_qc_pass=self.check_sample_read_count_above_threshold(sample_obj=sample_obj),
+            CG_ID_project=sample_obj.links[0].family.name,
+            Customer_ID_project=sample_obj.ticket_number,
+            application_tag=sample_obj.application_version.application.tag,
+            method_libprep=self.lims_api.get_sample_attribute(
+                lims_id=sample_obj.internal_id, key="pre_processing_method"
+            ),
+            method_sequencing="novaseq",
+            date_arrival=str(sample_obj.received_at),
+            date_libprep=str(sample_obj.prepared_at),
+            date_sequencing=str(sample_obj.sequenced_at),
+            selection_criteria=self.lims_api.get_sample_attribute(
+                lims_id=sample_obj.internal_id, key="selection_criteria"
+            ),
+            priority=self.lims_api.get_sample_attribute(
+                lims_id=sample_obj.internal_id, key="priority"
+            ),
+            region_code=self.lims_api.get_sample_attribute(
+                lims_id=sample_obj.internal_id, key="region_code"
+            ),
+            lab_code=self.lims_api.get_sample_attribute(
+                lims_id=sample_obj.internal_id, key="lab_code"
+            ),
+        )
+
+    def create_case_config(self, case_id: str, dry_run: bool) -> None:
+        case_obj = self.status_db.family(case_id)
+        samples: List[models.Sample] = [link.sample for link in case_obj.links]
+        case_config_list = [
+            self.get_sample_parameters(sample_obj=sample_obj).dict() for sample_obj in samples
+        ]
+        config_path = self.get_case_config_path(case_id=case_id)
+        if dry_run:
+            LOG.info("Dry-run, would have created config at path %s, with content:", config_path)
+            LOG.info(case_config_list)
+        json.dump(case_config_list, open(config_path, "w"), indent=4)
+        LOG.info("Saved config to %s", config_path)
+
+    def run_analysis(self, case_id: str, dry_run: bool) -> None:
+        if self.get_case_output_path(case_id=case_id).exists():
+            LOG.info("Found old output files, directory will be cleaned!")
+            shutil.rmtree(self.get_case_output_path(case_id=case_id), ignore_errors=True)
+
+        self.process.run_command(
+            [
+                "analyse",
+                "sarscov2",
+                "--config_case",
+                self.get_case_config_path(case_id=case_id).as_posix(),
+                "--outdir",
+                self.get_case_output_path(case_id=case_id).as_posix(),
+                self.get_case_fastq_dir(case_id=case_id).as_posix(),
+            ],
+            dry_run=dry_run,
+        )
+
+    def get_pipeline_version(self, case_id: str) -> str:
+        return str(
+            self.process.run_command(
+                [
+                    "--version",
```

Comment:
If this is used to determine MUTANTs version, it wont work

Reply:
i checked and it worked, but why??